lib/grids_page.dart

```diff
@@ -57,6 +57,10 @@ class GridsPage extends StatelessWidget {
                     },
                     child: const Text('Cancel'),
                   ),
+                  const SizedBox(height: 20),
+                  const Text('Current Grid:'),
+                  const SizedBox(height: 20),
+                  SelectGrid(grid: grid.value, currentGrid: grid)
                 ]),
```

Comment:
Instead of duplicating current grid, let's just highlight is with different background.
Let's do it in different PR, to simplify this PR and merge it earlier.

Reply:
Done in another PR